apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/Defendant/DefendantForm.tsx

```diff
@@ -76,26 +83,34 @@ const DefendantForm: React.FC<Props> = (props) => {
     defendantId: string,
     updatedDefendant: UpdateDefendant,
   ) => {
-    updateDefendantState(defendantId, updatedDefendant)
+    try {
+      updateDefendantState(defendantId, updatedDefendant)
 
-    if (workingCase.id) {
-      updateDefendant(workingCase.id, defendantId, updatedDefendant)
+      if (workingCase.id) {
+        updateDefendant(workingCase.id, defendantId, updatedDefendant)
+      }
+    } catch (error) {
```

Comment:
Consider removing this catch and handle the error in the useDefendant hook in a similar way as is done in the useCase hook.